packages/core/src/transport.ts

```diff
@@ -68,6 +71,24 @@ export class Batch<T> {
     }
   }
 
+  sizeInBytes(candidate: string) {
+    // Accurate byte size computations can degrade performances when there is a lot of events to process
+    if (!HAS_MULTI_BYTES_CHARACTERS.test(candidate)) {
+      return candidate.length
+    }
+
+    if (window.TextEncoder !== undefined) {
+      return new TextEncoder().encode(candidate).length
+    }
+
+    if (window.Blob !== undefined) {
+      return new Blob([candidate]).size
+    }
+
+    // tslint:disable-next-line no-bitwise
+    return ~-encodeURI(candidate).split(/%..|./).length
```

Comment:
I would be in favor of keeping only one of these fallbacks since both should work on our current browser target.
On IE11 Blob is faster, so I would keep only this one.